client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -85,16 +95,35 @@ class DesignTypeWithStoreStep extends Component {
 	};
 
 	renderChoice = ( choice ) => {
-		return (
-			<Card className="design-type-with-store__choice" key={ choice.type }>
-				<a className="design-type-with-store__choice-link"
-					href="#"
-					onClick={ this.handleChoiceClick( choice.type ) }>
-					{ choice.image }
-					<h2>{ choice.label }</h2>
-				</a>
-			</Card>
-		);
+		switch ( abtest( 'signupCopy' ) ) {
+			case 'newSignupCopy':
+				return (
```

Comment:
You'll probably improve this further, since this is just the initial version of the patch, but noting just in case: the common parts between the variations should not be duplicated if possible. In this case, you could even leave the whole HTML markup (from the new variation) the same - yes, including the `choice.description` (if it's empty, nothing will be shown). This tremendously improves the readability of the code - it's immediately visible what's the difference between the variations. And it's easier to spot/fix errors - you (possibly) don't need to do it in two places ;)